.github/workflows/integration_test.yaml

```diff
@@ -253,6 +255,25 @@ jobs:
           DURATION: ${{ inputs.chaos-status-duration }}
           EXPERIMENT_NAME: ${{ inputs.chaos-experiments }}
           TOTAL_CHAOS_DURATION: ${{ inputs.chaos-duration }}
+      - name: Import load test json.env file
+        if: ${{ inputs.load-test-enabled }}
+        run: echo "JSON_ENV=$(jq -c . < load_tests/env.json)" >> $GITHUB_ENV
+      - name: Import load test json.env file
+        if: ${{ inputs.load-test-enabled }}
+        run: echo ${{ fromJSON(env.JSON_ENV) }}
+      - name: Reconfigure iptables
+        if: ${{ inputs.load-test-enabled }}
+        run: |
+          # Workaround to fix IP tables so that microk8s is reacheable on port 80 during the load test execution
+          # https://github.com/canonical/lxd-cloud/blob/f20a64a8af42485440dcbfd370faf14137d2f349/test/includes/lxd.sh#L13-L23
+          sudo iptables -P FORWARD ACCEPT
+          curl localhost:80
```

Comment:
What's the purpose of this command?

Reply:
I was testing some things. Those lines have been removed.